prReview.js

```diff
@@ -0,0 +1,7 @@
+function findFolderByName(n) {
+	try {
+		return n === null ? null : (Folder.find({ name: n }) || Folder.create({ name: n }));
+	} catch(e) {
```

Comment:
**nitpick _(non-blocking)_:** Since "e" represent "error" let's write it in full, it's just five letter words. 🙂